.github/workflows/test.yml

```diff
@@ -23,26 +23,30 @@ jobs:
         cd ~/FFmpeg/
         ./prepare.sh  
       
-    - name: Setup Poetry
-      run: poetry install --extras vpf-samples        
-
+    - name: Setup pip
+      run: |
+        python3 -m venv /tmp/venv_vpf
+        source /tmp/venv_vpf/bin/activate
+        python3 -mpip install -r ./samples/requirements.txt    
+        
```

Comment:
add
```
on:
  schedule:
    - cron: '0 0 * * *'  # every day at midnight
```
in addition to current triggers to run the job daily? poetry broke due to an external dependency update.